demos/jtop-demo/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ description = "JTop - System monitor demo using the DSL module"
 
 dependencies {
     implementation(projects.tambouiToolkit)
+    implementation("com.github.oshi:oshi-core:6.9.2")
```

Comment:
You should put this in the libs.versions.toml file and use `libs.oshi.core` as a dependency notation.